packages/web3torrent/src/constants.ts

```diff
@@ -177,10 +177,7 @@ export {SINGLE_ASSET_PAYMENT_CONTRACT_ADDRESS};
 
 export const FUNDING_STRATEGY = process.env.FUNDING_STRATEGY === 'Direct' ? 'Direct' : 'Virtual';
 
-export const INITIAL_BUDGET_AMOUNT = utils.hexZeroPad(
-  utils.parseEther('0.000001').toHexString(),
-  32
-); // 1 Szabo (equivalent to 1000GB at 1 wei per byte)
+export const INITIAL_BUDGET_AMOUNT = utils.hexZeroPad(utils.parseEther('0.001').toHexString(), 32); //  (equivalent to 1GB at 1 wei per byte)
```

Comment:
It looks like this larger budget is making the `receivedPercentage`/`spendPercentage` too small to actually show up on the pie chart.

Reply:
Even at 1 szabo the percentage of funds locked up in a channel starts off really small which seems not to render in the pie chart.

Reply:
Yeah I set it to 1GB total initial budget (1000 times lower than it currently is), and was trying uploading a file of 100MB or so in an attempt to get visual confirmation of the data flow. But even then I don't see what I expect...

Reply:
> Yeah I set it to 1GB total initial budget (1000 times lower than it currently is), and was trying uploading a file of 100MB or so in an attempt to get visual confirmation of the data flow. But even then I don't see what I expect...

I think the change to `INITIAL_BUDGET_AMOUNT` might be incorrect here. `0.001 Eth` is 1000 szabo or 1000 times larger than the previous budget. If I set it to a `gwei` (`0.000000001 Eth`) I see the locked funds on the pie chart fairly quickly. (Unfortunately I'm running into some issues right now where I get stuck on connecting or I would show a screenshot :) )

Reply:
Using 
```
export const INITIAL_BUDGET_AMOUNT = utils.hexZeroPad(
  utils.parseEther('0.0000000001').toHexString(),
  32
); //  100 Mwei (or 100 mb) // TODO: Change this back
```
I see the pie chart look like I would expect (3.2Mwei would 3.2% of the budget)
![image](https://user-images.githubusercontent.com/1620336/83697009-b3f28a80-a5b2-11ea-847c-aa22b5097c46.png)

Reply:
Ah yes, it would have helped if I hadn't got confused between multiplication and division 😊